.github/workflows/packer-build-and-publish.yml

```diff
@@ -25,29 +25,81 @@ jobs:
           is_main=$([ -n "$(git branch -r --contains ${{ github.sha }} origin/main)" ] && echo true || echo false)
           echo "value=$is_main" >> "${GITHUB_OUTPUT}"
 
+  check-release:
```

Comment:
Could we simplify things by saying that a tag _always_ is a release? And then we don't have to handle the case where it's not found - that would be an error instead. Or maybe I'm missing a reason to do this.

Reply:
Yeah. So what's the `release_found` providing?

Reply:
Release found is used as a conditional on the build job, if a release is already found we don't need to rebuild and also modifies how the publish job works, updating the found release instead of creating a new one.